python_modules/dagster/dagster/core/definitions/sensor.py

```diff
@@ -517,3 +584,61 @@ def _fn(context):
     @property
     def asset_key(self):
         return self._asset_key
+
+
+@experimental
+class MultiJobSensorDefinition(ISensorDefinition):
+    """Define a sensor that targets multiple jobs.
```

Comment:
should add a section to the docs listing this experimental API

might also want to highlight that the idempotence checks against run_key use the sensor as the scope, not the sensor/job pairing.